tests/cephfs/snapshot_clone/cg_snap_test.py

```diff
@@ -0,0 +1,550 @@
+    6. Run 'ceph fs subvolume quiesce --release' to release quiesce set.
+    7. Repeat without --await option, verify cmd response has status as quiescing, wait for quiesced status,
+      run step 5 and 6.
+
+    Workflow2 -
+    Steps :
+    Workflow3 -
+    Steps:
+    1.
+    Workflow4 -
+    Steps:
+
+    Type - Longevity
+
+    Workflow5 -
+    Workflow6 -
+
+
+    Type - Negative
+    Workflow6 -
+    Workflow7 -
+
+    Type - Systemic
+    Workflow8 - s
+    Note :
+
+    Clean Up:
+    1.
+
+    """
+    try:
+        fs_util_v1 = FsUtilsv1(ceph_cluster)
+        cg_snap_util = CG_Snap_Utils(ceph_cluster)
+        cg_snap_io = CG_snap_IO(ceph_cluster)
+        config = kw.get("config")
+        clients = ceph_cluster.get_ceph_objects("client")
+        mgr_node = ceph_cluster.get_ceph_objects("mgr")[0]
+        build = config.get("build", config.get("rhbuild"))
+        fs_util_v1.prepare_clients(clients, build)
+        default_fs = config.get("fs_name", "cephfs")
+        fs_util_v1.auth_list(clients)
+
+        client1 = clients[0]
+        log.info("checking Pre-requisites")
+        if len(clients) < 1:
+            log.info(
+                f"This test requires minimum 1 client nodes.This has only {len(clients)} clients"
+            )
+            return 1
+
+        fs_details = fs_util_v1.get_fs_info(client1, fs_name=default_fs)
+        if not fs_details:
+            fs_util_v1.create_fs(client1, default_fs)
+
+        test_name_type = config.get("test_name", "all_tests")
+        test_functional = [
+            "cg_snap_test_func_1",
+        ]
+        test_negative = []
+        test_longevity = []
+        test_systemic = []
+        test_name_all = test_functional + test_negative + test_longevity + test_systemic
+
+        test_dict = {
+            "all_tests": test_name_all,
+            "functional": test_functional,
+            "longevity": test_longevity,
+            "systemic": test_systemic,
+            "negative": test_negative,
+        }
+        if test_name_type in test_dict:
+            test_list = test_dict[test_name_type]
+        else:
+            test_list = [test_name_type]
+
+        # Setup
+        subvolumegroup = {
+            "vol_name": default_fs,
+            "group_name": "subvolgroup_cg",
+        }
+        fs_util_v1.create_subvolumegroup(client1, **subvolumegroup)
+        for i in range(1, 3):
+            subvolume = {
+                "vol_name": default_fs,
+                "subvol_name": f"sv_def_{i}",
+                "size": "6442450944",
+            }
+            fs_util_v1.create_subvolume(client1, **subvolume)
+        for i in range(1, 3):
+            subvolume = {
+                "vol_name": default_fs,
+                "subvol_name": f"sv_non_def_{i}",
+                "group_name": "subvolgroup_cg",
+                "size": "6442450944",
+            }
+            fs_util_v1.create_subvolume(client1, **subvolume)
+        qs_sets = [
```

Comment:
lets have a minimum of 5 subvolumes in each quiesce set as per PM recommendations.

Reply:
As the 5 subvol update came later, its being considered for coding this week, its already being implemented, will validate and push changes by tomorrow.